pkg/actions/irsa/get_test.go

```diff
@@ -24,8 +24,8 @@ var _ = Describe("Get", func() {
 		irsaManager = irsa.New("my-cluster", fakeStackManager, nil, nil)
 	})
 
-	When("no options are specified", func() {
-		It("returns all service accounts", func() {
+	When("no error occurs", func() {
```

Comment:
Most of the logic that these tests covered is now in `GetIAMServiceAccounts`, with test coverage in `iam_test.go`.